### PR TITLE
Uploaded Tuple.java and TupleDesc.java

### PR DESCRIPTION
**Pull request by Rithesh17**
#1 

I have successfully completed the code of the two files. I have completed the entire coding myself, though Manish helped me in the debugging part.
It didn't take long for me to code the Tuple file. I could finish it in a few hours. TupleDesc file too didn't take much time, except for the Iterator code. I took a lot of time in finding out the code. But in the end, I could do it.

### CODE REVIEW

**Comment by sbshah97** [comment]
Your teammates have to review the code for you and after you get three approvals merge it!

**Comment by sbshah97** [comment]
Code looks fine to me!

**Review by manishsbhoopalam — APPROVED** [review]

**Review by Priiyam — APPROVED** [review]

**Pull request merged**
